cluster_opts_test.go

```diff
@@ -28,7 +28,6 @@ func TestClusterDefaults(t *testing.T) {
 	f := newFixture(t, 1)
 	c, err := NewCluster(f.ClusterNodes(), f.PrimaryChecker)
 	require.NoError(t, err)
-	defer func() { require.NoError(t, c.Close()) }()
```

Comment:
Why Close is removed here and in other tests?

Reply:
It has been removed due to pointlessness. The only reason it has been called is due to expectation in test suite generator. AFAIK `hasql` does not oblige user to call `Close` at any point.

As expectation has been removed so were all `Close` calls in tests. `Close` method has its own test function, so I believe all test cases relevant to cluster teardown must be placed there.